docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module

```diff
@@ -735,3 +739,36 @@ function cgov_addl_lib_default_value_function(EntityInterface $entity, FieldDefi
     ['value' => 'None'],
   ];
 }
+
+
+function cgov_core_redirect_insert($entity) {
+  $moduleHandler = \Drupal::service('module_handler');
+  if ($moduleHandler->moduleExists('purge')) {
+    // host is getting set from sites/settings/cgov_core.settings.php.
+    $host = Settings::get('cgdp.cache_url_host');
```

Comment:
Not to be overly nit-picky, but `$host` is set right there on line 747.  The _value_ for `cgdp.cache_url_host` is computed in `cgov_core.settings.php`.